### PR TITLE
Fixes 763: update properties deprecated in Firefox 69

### PR DESCRIPTION
**Pull request by mgillespy**

### CODE REVIEW

File: src/lib/themes.js [hunk 1]
```diff
@@ -166,11 +166,25 @@ export const normalizeTheme = (data = {}) => {
...
       theme.images.additional_backgrounds = [background];
     }
   }
+  // Fx69 update - deprecate accentcolor
+  if (theme.colors.accentcolor) {
+    theme.colors.frame = theme.colors.accentcolor;
+  }
+ // Fx69 update - deprecate textcolor
+  if (theme.colors.textcolor) {
+    theme.colors.frame = theme.colors.textcolor;
```
Comment (nt1m) [inline src/lib/themes.js]: You probably meant:
```suggestion
    theme.colors.tab_background_text = theme.colors.textcolor;
```

File: src/lib/themes.js [hunk 2]
```diff
@@ -166,11 +166,25 @@ export const normalizeTheme = (data = {}) => {
   }
 
   if (images.headerURL) {
-    const background = normalizeThemeBackground(images.headerURL);
+  // Fx69 update - deprecate headerURL
```
Comment (nt1m) [inline src/lib/themes.js]: small nit: strange indentation
```suggestion
    // Fx69 update - deprecate headerURL
```

File: src/lib/themes.js [hunk 3]
```diff
@@ -166,11 +166,25 @@ export const normalizeTheme = (data = {}) => {
...
+    const background = normalizeThemeBackground(images.theme_frame);
     if (background) {
       theme.images.additional_backgrounds = [background];
     }
   }
+  // Fx69 update - deprecate accentcolor
+  if (theme.colors.accentcolor) {
+    theme.colors.frame = theme.colors.accentcolor;
+  }
+ // Fx69 update - deprecate textcolor
```
Comment (nt1m) [inline src/lib/themes.js]: Ditto :)

File: src/lib/themes.js [hunk 4]
```diff
@@ -166,11 +166,25 @@ export const normalizeTheme = (data = {}) => {
...
+  if (theme.colors.textcolor) {
+    theme.colors.frame = theme.colors.textcolor;
+  }
+ // Fx69 update - deprecate toolbar_text
```
Comment (nt1m) [inline src/lib/themes.js]: Here too.

File: src/lib/themes.js (hunk 3 above)
Comment (mgillespy) [reply src/lib/themes.js]: This one looks ok to me?

Comment (mgillespy) [reply src/lib/themes.js]: Oh, I see... do you mean indentation again? Got it.

**Comment by rpl** [comment]
@ntim we could close this one as it is included and superseded by #819 (I can't close it, I don't have the needed privileged on this repo).

**Comment by nt1m** [comment]
@rpl I believe @caitmuenster has the right permissions.

**Comment by caitmuenster** [comment]
On it! Thank you @rpl (and @mgillespy for the original PR!).

**Pull request closed without merging**